app/services/statements/query.rb

```diff
@@ -54,6 +54,10 @@ def where_updated_since(updated_since)
     def where_state_is(state)
       return if ignore?(filter: state)
 
+      unless extract_conditions(state).all? { |s| Statement.state_machine.states.keys.map(&:to_s).include?(s) }
```

Comment:
can we drop this error as we don't expose filtering by state in the API, it's only internally for output fee true purposes